src/pd/pd.rs

```diff
@@ -486,20 +595,29 @@ impl<T: PdClient> Runner<T> {
         self.is_hb_receiver_scheduled = true;
     }
 
+    fn maybe_handle_reconnect(&mut self) {
+        if self.invalidate_cache.swap(false, Ordering::Relaxed) {
```

Comment:
I prefer `Ordering::SeqCst` to save other's brain.

Reply:
`Relaxed` means that there is no memory ordering problem, so I think `Relaxed` is more clear.